src/utils/types.ts

```diff
@@ -2,3 +2,6 @@ export const isTruthy = <T>(item: T): boolean => !!item;
 
 export const hasElements = <T>(array?: T[] | null): boolean =>
   isTruthy(array?.length);
+
+export const notEmpty = <T>(value: T | null | undefined): value is T =>
+  value !== null && value !== undefined;
```

Comment:
[OOS, Future, musing] I wonder if these belong in a better place than `utils/types` (honestly not even sure what this filename means in this usage), I know we have `utils/filters`, though these also have usages outside of just being used with filters. `utils/utils`? :p

Reply:
It ended up in here because it's related to type safety - it is more versatile than just a filter IMO. I'll leave it here for now and we can see if we end up needing it for anything else - that might guide where its final home ends up being.